federator/src/lib/logs.ts

```diff
@@ -1 +1,86 @@
-export class Logs {}
+import log4js from 'log4js';
+import { LogWrapper } from './logWrapper';
+import logConfig from '../../config/log-config.json';
+
+export const LOGGER_CATEGORY_TEST_INTEGRATION = 'test';
+export const LOGGER_CATEGORY_TEST_FEDERATOR = 'FEDERATOR';
+export const LOGGER_CATEGORY_TEST_FEDERATOR_NFT = 'NFT FEDERATOR';
+export const LOGGER_CATEGORY_HEARTBEAT = 'HEARTBEAT';
+export const LOGGER_CATEGORY_FEDERATOR = 'Federators';
```

Comment:
I think we should choose one casing standard? maybe all names should be in caps?

Reply:
Agreed